boards/arm/pinetime_devkit0/pinetime_devkit0.dts

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * DTS for Pine64 PineTime DevKit-0 board
+ * Further info: https://wiki.pine64.org/index.php/PineTime
+ *
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832_qfaa.dtsi>
+
+/ {
+	model = "Pine64 PineTime DevKit0";
+	compatible = "pine64,pinetime-devkit0";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	aliases {
+		led0 = &blled0;	/* backlight low */
+		led1 = &blled1;	/* backlight mid */
+		led2 = &blled2;	/* backlight high */
+		led3 = &statusled;	/* status led, may be not populated */
+		sw0 = &key_in;  /* key in */
+		sw1 = &key_out;  /* key out */
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blled0: bl_led_0 {
+			gpios = <&gpio0 14 0>;
+			label = "Backlight LED 0";
+		};
+		blled1: bl_led_1 {
+			gpios = <&gpio0 22 0>;
+			label = "Backlight LED 1";
+		};
+		blled2: bl_led_2 {
+			gpios = <&gpio0 23 0>;
+			label = "Backlight LED 2";
+		};
+		statusled: led_3 {
+			gpios = <&gpio0 27 0>;
+			label = "Status Led";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		key_in: button_0 {
+			gpios = <&gpio0 13 0>;
+			label = "Key in";
+		};
+		key_out: button_1 {
+			gpios = <&gpio0 15 0>;
+			label = "Key out";
+		};
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "okay";
+	tx-pin = <11>;
+	rx-pin = <30>;
+};
+
+&i2c0 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+	sda-pin = <6>;
+	scl-pin = <7>;
+	clock-frequency = <I2C_BITRATE_FAST>; /* 400KHz */
+
+	/* BOSCH BMA421 Triaxial Acceleration Sensor (1000KHz) */
+	bma421: bma421@18 {
+		compatible = "bosch,bma4xx";
+		reg = <0x18>;
+		label = "BMA421";
+		int1-gpios = <&gpio0 8 0>;
+	};
+
+	/* TianYiHeXin HRS3300 PPG Hear Rate Sensor (800KHz) */
+	hrs3300: hrs3300@44 {
+		compatible = "tian-yi-he-xin,hrs3300";
+		reg = <0x44>;
+		label = "HRS3300";
+	};
+
+	/* Hynitron CST816S Capacitive Touch Controller (400KHz) */
+	cst816s: cst816s@15 {
+		compatible = "hynitron,cst816s";
+		reg = <0x15>;
+		label = "CST816S";
+		irq-gpios = <&gpio0 28 0>;
+		rst-gpios = <&gpio0 10 0>;
```

Comment:
> Aren't these two pins active low?

Checked with [working firmware](https://github.com/JF002/Pinetime) and yes, they are active low.

Reply:
Then the `GPIO_ACTIVE_LOW` flag should be used instead of `0` as the third item in both those arrays.
And similarly for other active low pins referred in this file.